tox.ini

```diff
@@ -125,7 +125,6 @@ description = Build only the API documentation
 
 commands =
     pydoctor \
-    --project-name=pydoctor \
```

Comment:
Should this option be removed now that the metadata change has been split off?

Reply:
True! Thanks.